src/components/Autosuggest/Autosuggest.stories.mdx

```diff
@@ -9,16 +9,34 @@ import * as stories from "./Autosuggest.stories.tsx";
 The NYPL Design System does not have its own `Autosuggest` component. Instead,
 we are currently using the
 [react-autosuggest](https://www.npmjs.com/package/react-autosuggest) package.
+We recommend using this package in your application and wrapping it in a custom
+component. The NYPL Design System does have custom styling that targets the
+classes from the `react-autosuggest` package, so the look and feel will be consistent.
+
+In the following previews, you'll notice that there is a lot of white space underneath
+every input element. The suggestion dropdown hovers over the page's content and
+a min-height is added to each block to better display the dropdown.
```

Comment:
Please update to read:   "In the following previews, you'll notice that there is a lot of white space underneath each input element. Since the suggestion dropdown will appear _on top of_ page content, we've added a min-height to make it easier to see the dropdown.  This is for demonstration purposes only and you will not need to do this in your application."

Reply:
Sure.